restli-tools/src/test/extensions/validCase/BazExtensions.pdl

```diff
@@ -0,0 +1,3 @@
+record BazExtensions includes Baz {
+  testField: DummyKey
```

Comment:
I would leave a comment saying that this is valid because extension annotations are optional for 1-to-1 relations.